docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 web:
   build: .
-  command: mix phoenix.server
+  command: service nginx start && mix phoenix.server
```

Comment:
I don't think you need this if you want the default `CMD` from the `Dockerfile` to run.